### ISSUE TITLE
invalid argument  - issue

### ISSUE THREAD

**Issue by suresh-s** [liuch37/pan-pytorch#6]
Resnet 101 training is working now, but i am getting following error, but still training is happening

Below is the error trace.  Could you tell me what is the issue ?

**THCudaCheck FAIL file=/pytorch/aten/src/THC/THCGeneral.cpp line=383 error=11 : invalid argument
/root/.virtualenvs/panplus_env/lib/python3.6/site-packages/torch/nn/functional.py:2539: UserWarning: Default upsampling behavior when mode=bilinear is changed to align_corners=False since 0.4.0. Please specify align_corners=True if the old behavior is desired. See the documentation of nn.Upsample for details.
  "See the documentation of nn.Upsample for details.".format(mode))
(1/209) LR: 0.001000 | Loss: 1.758 | Loss (text/kernel/emb): 0.540/0.262/0.955 | IoU (text/kernel): 0.195/0.283**

**Comment by suresh-s** [comment]
I have fixed by using pytorch 1.0.0 and torchvision 0.2.0
